language/gpt3/megatron/README.md

```diff
@@ -0,0 +1,123 @@
+# GPT-3 Reference Implementation
+
+## Setup Instructions
+
+```bash
+WORK_DIR=$PWD
+# Create Environment (conda)
+conda create -n llm python=3.9 -y
+conda activate llm
+```
+
+### Download repositories
+```bash
+cd $HOME
+git clone --recurse-submodules https://github.com/mlcommons/inference.git --depth 1
+git clone https://github.com/NVIDIA/apex.git
+git clone https://github.com/mlcommons/training.git --branch inference-megatron
+cd $HOME/apex
+git checkout -b language 2d8302a6c12e202f7b40b13a43daa95f326fd0ea
+```
+
+
+### install requirements
+```bash
+pip install torch==1.13.0 torchvision==0.14.0 datasets evaluate accelerate simplejson nltk rouge_score pybind11 Ninja numpy==1.19.5 sentencepiece
+pip install git+https://github.com/NVIDIA/mlperf-common.git
+pip install git+https://github.com/mlperf/logging.git
+
+```
+
+#### install apex
+For `pip >= 23.1`
+```bash
+cd $HOME/apex
+pip install -v --disable-pip-version-check --no-cache-dir --no-build-isolation --config-settings "--build-option=--cpp_ext" --config-settings "--build-option=--cuda_ext" ./
```

Comment:
I suggest run `pip wheel` then `pip install *.whl` to cache the wheel. This step takes a long time to run